core/tests/puppeteer-acceptance-tests/specs/curriculum-admin/create-publish-unpublish-and-delete-topic-and-skill.spec.ts

```
@@ -51,59 +52,84 @@ describe('Curriculum Admin', function () {
+      'Test Skill 1',
+      'Test Subtopic 1',
+      'Test Topic 1'
+    );
+    await curriculumAdmin.addSkillToDiagnosticTest(
+      'Test Skill 1',
+      'Test Topic 1'
+    );
+
+    // Publish topic.
+    await curriculumAdmin.publishDraftTopic('Test Topic 1');
+    await curriculumAdmin.expectToBeInTopicAndSkillsDashboardPage();
+    await curriculumAdmin.expectTopicToBePublishedInTopicsAndSkillsDashboard(
+      'Test Topic 1',
+      0, // Story added.
+      1, // Subtopic added.
+      1 // Skill added.
+    );
+
+    await curriculumAdmin.openTopicEditor('Test Topic 1');
+    await curriculumAdmin.expectUnpublishTopicButtonToBeVisible();
+  });
+
+  it('should be able to create a skill', async function () {
+    await curriculumAdmin.navigateToTopicAndSkillsDashboardPage();
+    await curriculumAdmin.navigateToSkillsTab();
+
+    await curriculumAdmin.clickOnCreateNewSkillButtonInSkillDashboard();
+    await curriculumAdmin.fillSkillDetailsInNewSkillModal(
+      'Test Skill 2',
+      'Test Review Material'
+    );
+    const newPage = await curriculumAdmin.clickOnElementAndGetNewPage('Save');
+    await curriculumAdmin.expectToBeInSkillEditorPage(newPage);
+  });
+
+  it('should be able to unpublish a topic', async function () {
+    await curriculumAdmin.unpublishTopic('Test Topic 1');
+    await loggedInUser.expectTopicLinkReturns404('test-topic-one');
+  });
+
+  it('should be able to delete a topic', async function () {
+    await curriculumAdmin.deleteTopic('Test Topic 1');
+    await curriculumAdmin.expectTopicNotInTopicsAndSkillDashboard(
+      'Test Topic 1'
+    );
+  });
+
+  it('should be able to delete a skill', async function () {
+    // User must remove all questions from the skill before deleting it.
+    await curriculumAdmin.removeAllQuestionsFromTheSkill('Test Skill 1');
+    await curriculumAdmin.deleteSkill('Test Skill 1');
+    await curriculumAdmin.expectSkillNotInTopicsAndSkillsDashboard(
+      'Test Skill 1'
+    );
```

The part of `The respective links for skill should return a 404 error` is missing.

---

@chris7716 Done. For this step, I have added steps in line 132 and 133. Let me know if that's okay.